scripts/generate_tilesets.py

```diff
@@ -467,16 +470,18 @@ def main(cfg_file_path):
                     logger.info(f'A seed of {seed} produces a good repartition of the labels.')
                     SEED = seed
                     break
-                
-                if seed == 10:
-                    logger.warning('No good seed found between 0 and 10. The user should set a seed manually.')
+                elif ok_split > max_split:
```

Comment:
Before, the code simply stopped searching and said it did not find any good seed. Then it continued with the last tested seed and you were supposed to manually search for a better seed.
Now, it at least gives you the best tested seed and continues with it, instead of just taking the last one.